pkgs/applications/video/kodi/addons/sendtokodi/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, buildKodiAddon, fetchFromGitHub, addonUpdateScript, inputstreamhelper }:
+
+buildKodiAddon rec {
+  pname = "sendtokodi";
+  namespace = "plugin.video.sendtokodi";
+  version = "0.9.557";
+
+  src = fetchFromGitHub {
+    owner = "firsttris";
+    repo = "plugin.video.sendtokodi";
+    rev = "v${version}";
+    hash = "sha256-Ga+9Q7x8+sEmQmteHbSyCahZ/T/l28BAEM84w7bf7z8=";
+  };
+
+  propagatedBuildInputs = [
+    inputstreamhelper
+  ];
+
+  passthru = {
+    pythonPath = "lib";
+  };
+
+  meta = with lib; {
+    homepage = "https://github.com/firsttris/plugin.video.sendtokodi";
+    description = "Plays various stream sites on Kodi using youtube-dl";
+    license = licenses.mit;
+    maintainers = teams.kodi.members ++ [ maintainers.pks ];
```

Comment:
Note that I've proactively added the Kodi members as maintainers of this package. I'm happy to drop you folks though in case you don't want to be listed here and take sole ownership of this package.

Reply:
you were correct to add the team 👍